### ISSUE TITLE
Collections in collections page in test tier - Clicking source link gives a 404 error page 

### ISSUE THREAD

**Issue by pgundluru** [ImagingDataCommons/IDC-WebApp#1038]
https://testing-portal.canceridc.dev/collections/10.7937/3CX3-S132 

![image](https://user-images.githubusercontent.com/85526819/186965133-5e51761c-0ff1-4569-a3e3-ed3f2a8c1716.png)

**Comment by fedorov** [comment]
Those are DOIs, they should be prefixed with "dx.doi.org".

**Comment by s-paquette** [comment]
Looks like we now have both DOIs *and* source URLs, which wasn't a behavior we had previously in ETL. @bcli4d was this an intentional change? Previously a source URL was just for collections with no DOI; now some of them have both. I don't mind needed to adjust the code, but I might need a reminder/run down on what to expect in these 2 columns now.

**Comment by fedorov** [comment]
That is something we discussed with Bill. I thought what we had previously was very confusing - one had to check both the URL and DOI columns to get to any documentation about the collection. I believe what we discussed with Bill was that for those collections that have DOI but do not have the URL, the URL would be dx.doi.org/+DOI.

Taking a look at the result of the query below, this is what Bill implemented, which I think is very helpful - `source_url` is now always non-empty.

```sql
SELECT distinct(source_doi), source_url FROM `idc-pdp-staging.idc_v11.dicom_all` 
```
